cocos/2d/assembler/label/letter-font.ts

```diff
@@ -72,9 +72,9 @@ export const letterFont = js.mixin(bmfontUtils, {
         return fontFamily;
     },
 
-    _updateLabelInfo (comp) {
+    _updateLabelInfo (comp: Label) {
         shareLabelInfo.fontDesc = this._getFontDesc();
-        shareLabelInfo.color = comp.color;
+        shareLabelInfo.color.set(comp.color);
```

Comment:
It changes the logic. Please make sure it is demanded.

Reply:
Please see the comment I replied above.